ts/screens/authentication/SpidInformationScreen.tsx

```diff
@@ -55,65 +68,89 @@ class SpidInformationScreen extends React.Component<Props, never> {
+            <H1>{I18n.t("authentication.spid_information.contentTitle")}</H1>
 
-            {this.getValueContent(
-              I18n.t("authentication.spid_information.point2-value"),
-              I18n.t("authentication.spid_information.point2-content")
-            )}
+            <View spacer={true} large={true} />
+            <Text>{I18n.t("authentication.spid_or_cie")}</Text>
+          </View>
+          <Tabs
+            tabBarUnderlineStyle={styles.tabBarUnderline}
+            tabContainerStyle={styles.tabBarContainer}
+          >
+            <Tab heading={I18n.t("authentication.spid")}>
+              <Content>
+                <Markdown>
+                  {I18n.t("authentication.spid_information.spid")}
+                </Markdown>
+                <Grid>
+                  {this.getValueContent(
+                    I18n.t("authentication.spid_information.point1-value"),
+                    I18n.t("authentication.spid_information.point1-content")
+                  )}
 
-            {this.getValueContent(
-              I18n.t("authentication.spid_information.point3-value"),
-              I18n.t("authentication.spid_information.point3-content")
-            )}
+                  {this.getValueContent(
+                    I18n.t("authentication.spid_information.point2-value"),
+                    I18n.t("authentication.spid_information.point2-content")
+                  )}
 
-            {this.getValueContent(
-              I18n.t("authentication.spid_information.point4-value"),
-              I18n.t("authentication.spid_information.point4-content")
-            )}
-          </Grid>
-          <View spacer={true} extralarge={true} />
-        </Content>
+                  {this.getValueContent(
+                    I18n.t("authentication.spid_information.point3-value"),
+                    I18n.t("authentication.spid_information.point3-content")
+                  )}
 
-        <View footer={true}>
-          <Button block={true} primary={true} onPress={this.browseToLink}>
-            <Text>{I18n.t("authentication.spid_information.knowMore")}</Text>
-          </Button>
-        </View>
+                  {this.getValueContent(
+                    I18n.t("authentication.spid_information.point4-value"),
+                    I18n.t("authentication.spid_information.point4-content")
+                  )}
+                </Grid>
+                <View spacer={true} extralarge={true} />
+              </Content>
+              <View footer={true}>
+                <Button
+                  block={true}
+                  primary={true}
+                  onPress={() => this.browseToLink("https://www.spid.gov.it")}
+                >
+                  <Text>
+                    {I18n.t("authentication.spid_information.knowMore")}
+                  </Text>
+                </Button>
+              </View>
+            </Tab>
+            <Tab heading={I18n.t("authentication.cie")}>
```

Comment:
When selected the title goes in 2 rows (iPhone X)
<img src="https://user-images.githubusercontent.com/822471/67314881-c3323580-f505-11e9-92e6-bec4687c81df.gif" height="600" />